lib/wallet/rpc.js

```diff
@@ -1877,31 +1877,44 @@ class RPC extends RPCBase {
   }
 
   async sendOpen(args, help) {
-    if (help || args.length < 1 || args.length > 2)
-      throw new RPCError(errs.MISC_ERROR, 'sendopen "name" ( force )');
+    if (help || args.length < 1 || args.length > 3)
+      throw new RPCError(errs.MISC_ERROR, 'sendopen "name" ( force ) ( account )');
 
     const wallet = this.wallet;
     const valid = new Validator(args);
     const name = valid.str(0);
     const force = valid.bool(1, false);
+    let account = valid.str(2, '');
+
+    if (account === '')
+      account = 'default';
```

Comment:
pretty sure you can combine these right? `valid.str(2, 'default')` ?

Reply:
The difference is when an empty string is passed. And this is exactly how other rpc functions are written

Reply:
not always? https://github.com/bcoin-org/bclient/blob/master/bin/bwallet-cli#L39

Reply:
but I see, its always this way in rpc.js

Reply:
Yeah, I mean empty strings become `'default'` for the other rpc functions

Reply:
Frankly, based on somewhat shallow review, I think `force` is actually unused, not sure what it was ever for, or if it's still supposed to even be there...?

Reply:
Im resolving this convo if thats ok :)